src/main/java/com/faforever/client/game/CreateGameController.java

```diff
@@ -470,4 +482,14 @@ boolean selectMap(String mapFolderName) {
   void setOnCloseButtonClickedListener(Runnable onCloseButtonClickedListener) {
     this.onCloseButtonClickedListener = onCloseButtonClickedListener;
   }
+
+  public void onMapFilterButtonClicked() {
+    mapFilterButton.setSelected(mapFilterController.getFilterAppliedProperty().getValue());
```

Comment:
Why set selected on clicking the button if you have the listener on the filter applied property?

Reply:
Because pressing the button changes the "selected" state.
This is done to let the user know that the filter has been applied even though the map filter popup has been closed.

Reply:
Gotcha